jgrapht-ext/src/test/java/org/jgrapht/ext/DOTImporterTest.java

```diff
@@ -156,9 +156,6 @@ public void updateVertex(
        importer.read(input, result);
 
        Assert.assertEquals(expected.toString(), result.toString());
-
```

Comment:
It would not work since the importer creates new instances for the edges. The two graphs have different edge sets.